x/authz/client/cli/tx.go

```diff
@@ -203,6 +205,20 @@ Examples:
 				return err
 			}
 
+			rules, err := cmd.Flags().GetString(FlagAuthzRules)
+			if err != nil {
+				return err
+			}
+
+			if rules != "" {
+				contents, err := os.ReadFile(rules)
```

Comment:
It should validate against allowed list of inputs.

Suppose if the app developers allow only `allowed_recipients`, `max_amount` for bankMsg.Send type, it should only accept those inputs. You should check the same in the msg_server implementation too. msg.Validate() may be.